internal/cmd/stack/run_discard.go

```diff
@@ -0,0 +1,58 @@
+package stack
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/shurcooL/graphql"
+	"github.com/urfave/cli/v2"
+
+	"github.com/spacelift-io/spacectl/internal/cmd/authenticated"
+)
+
+func runDiscard() cli.ActionFunc {
+	return func(cliCtx *cli.Context) error {
+		stackID := cliCtx.String(flagStackID.Name)
+
+		var mutation struct {
+			RunDiscard struct {
+				ID string `graphql:"id"`
+			} `graphql:"runDiscard(stack: $stack, run: $run)"`
+		}
+
+		variables := map[string]interface{}{
+			"stack": graphql.ID(stackID),
+			"run":   graphql.ID(cliCtx.String(flagRun.Name)),
+		}
+
+		ctx := context.Background()
+
+		var requestOpts []graphql.RequestOption
+		if cliCtx.IsSet(flagRunMetadata.Name) {
+			requestOpts = append(requestOpts, graphql.WithHeader(UserProvidedRunMetadataHeader, cliCtx.String(flagRunMetadata.Name)))
+		}
+
+		if err := authenticated.Client.Mutate(ctx, &mutation, variables, requestOpts...); err != nil {
+			return err
+		}
+
+		fmt.Println("You have successfully discarded a deployment")
+
+		fmt.Println("The live run can be visited at", authenticated.Client.URL(
```

Comment:
The copy is misleading, at this stage it's no longer a "live" run.

Reply:
Good catch!